Source/Core/InputCommon/ControllerInterface/WGInput/WGInput.h

```diff
@@ -0,0 +1,222 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#pragma warning(push)
+#pragma warning(disable : 4265)
```

Comment:
Please add a comment describing what the warning is so everyone doesn't have to Google it :)